kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/KafkaDataBuilder.java

```diff
@@ -52,7 +53,9 @@ public static Field buildKafkaDataField(String kafkaDataFieldName) {
                 .setMode(com.google.cloud.bigquery.Field.Mode.NULLABLE);
         Field insertTimeField = insertTimeBuilder.build();
 
-        return Field.newBuilder(kafkaDataFieldName, LegacySQLTypeName.RECORD,
+        return Field.newBuilder(
+                FieldNameSanitizer.sanitizeName(kafkaDataFieldName),
```

Comment:
Why are we unconditionally sanitizing here but conditionally sanitizing the key field in `SchemaManager::getRegularSchemaFields`?

Also, could we maybe move this logic into `SchemaManager:getRegularSchemaFields` just to avoid spreading it across so many different places and implementing it in slightly different ways?

Reply:
Ah, good catch. Yes, moving this logic out into `getRegularSchemaFields` would consolidate the logic there a bit more. Will do.